.github/ISSUE_TEMPLATE/project-onboarding.md

```diff
@@ -38,11 +38,11 @@ A "Project Contribution Agreement" must be completed and any existing trademarks
 
 - [ ] Move your project to its own separate neutral GitHub organization. This will make it transferable to the CNCF's GitHub Enterprise account. If it's already in a GHE account, you will need to remove it from that first.
 - [ ] Accept the invite to join the CNCF GitHub Enterprise account. We'll then add `thelinuxfoundation` as an organization owner to ensure neutral hosting of your project.
-- [ ] Migrate your Slack channels (if any) to the [Kubernetes or CNCF Slack workspace](https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another). CNCF staff can help.
+- [ ] Migrate your Slack channels (if any) to the [Kubernetes or CNCF Slack workspace](https://slack.com/help/articles/217872578-Import-data-from-one-Slack-workspace-to-another). CNCF staff can help. If you use Discord, create a channel in Slack and provide a link to it.
 - [ ] Join the [#maintainers-circle](https://app.slack.com/client/T08PSQ7BQ/C014YQ8CDCG) Slack channel.
 - [ ] Transfer [your domain(s) to the CNCF](https://jira.linuxfoundation.org/plugins/servlet/desk/portal/2/create/63?requestGroup=19) if they exist. The "LF Stakeholder email" is <projects@cncf.io>. The "Project" is CNCF.
 - [ ] Submit a [pull request](https://github.com/cncf/artwork) with your artwork if it exists.
-- [ ] Transfer website analytics to <projects@cncf.io> if they exist. CNCF staff can help.
+- [ ] Transfer website analytics to <projects@cncf.io> if they exist. Once you have Service Desk access you can open a ticket.
```

Comment:
For the past three projects we seem to have been ok using the project's onboarding issue directly, rather than require that the maintainers create a service desk request. This is because it was a simple org transfer of GA properties. If you're ok with handling such a transfer via the onboarding issue, I'm ok with it too.

Other cases might be a bit more involved (e.g., they're not using GA and/or they'd like us to create a GA property for them), then it would be best for projects to create a Service Desk request in that case.